wearipedia/devices/polar/vantage_synthetic.py

```diff
@@ -0,0 +1,297 @@
+                "Date": d.strftime("%d-%m-%Y"),
+                "Start time": str(pd.to_datetime(activity["startDate"][:19])).split(
+                    " "
+                )[1]
+                + " "
+                + suffix,
+                "Duration": str(int(duration / 3600000)).zfill(2)
+                + ":"
+                + str(int(duration / 60000)).zfill(2)
+                + ":"
+                + str(int(duration % 1000))[:2].zfill(2),
+                "Total distance (mi)": np.round(distance, 2)
+                if activity["sportId"] != 15
+                else None,
+                "Average heart rate (bpm)": activity["hrAvg"],
+                "Average speed (mi/h)": np.round(
+                    distance / (activity["duration"] / 3600000), 1
+                )
+                if activity["sportId"] != 15
+                else None,
+                "Max speed (mi/h)": np.round(
+                    distance / (activity["duration"] / 3600000)
+                )
+                if activity["sportId"] != 15
+                else None,
+                "Average pace (min/mi)": distance / (activity["distance"] / 3600000)
+                if activity["sportId"] != 5
+                else None,
+                "Max pace (min/mi)": distance / (activity["distance"] / 3600000)
+                if activity["sportId"] != 15
+                else None,
+                "Calories": activity["calories"],
+                "Fat percentage of calories(%)": 55,
+                "Carbohydrate percentage of calories(%)": 45,
+                "Protein percentage of calories(%)": 0,
+                "Average cadence (rpm)": 0,
+                "Average stride length (in)": 0,
+                "Running index": 0,
+                "Training load": 0,
+                "Ascent (ft)": 0,
+                "Descent (ft)": 0,
+                "Average power (W)": 0,
+                "Max power (W)": 0,
+                "Notes": " ",
+                "Height (ft in)": "5 11",
+                "Weight (lbs)": 119.0,
+                "HR max": int(max_syn_hrs("")),
+                "HR sit": int(min_syn_hrs("")),
+                "VO2max": int(random_vo2Max("")),
+                "Unnamed: 29": None,
+            }
+        ]
+
+        # Create a a dictionary for the training session by id on the current date
+        temp = np.random.randint(50, 90)
+
+        # Our sample time is 1 second, so we need to loop through the duration of the activity
+        for i in range(duration // 1000):
+            curr_id_traning.append(
+                {
+                    "Sample rate": 1,
+                    "Time": str(i // 3600).zfill(2)
+                    + ":"
+                    + str(i // 60).zfill(2)
+                    + ":"
+                    + str(i % 60).zfill(2),
+                    "HR (bpm)": int(random_hr_avg("")),
+                    "Speed (mi/h)": np.round(
+                        distance / (activity["duration"] / 3600000), 1
```

Comment:
It looks like your speed and pace variables are constant for the activity. Could you try to randomize it?